plugins/scanners/fossid/src/main/kotlin/FossIdNamingProvider.kt

```diff
@@ -37,7 +37,8 @@ import org.apache.logging.log4j.kotlin.logger
  * * **currentTimestamp**: The current time.
  * * **deltaTag** (scan code only): If delta scans are enabled, this qualifies the scan as an *origin* scan or a *delta*
  * scan.
- * * **branch**: The branch name (revision) to scan.
+ * * **branch**: The branch name (revision) to scan. Characters not matching `[a-zA-Z0-9-_]` are replaced with
```

Comment:
Also here, I wonder whether hard-coding the regex really makes sense. Maybe just say a bit lax "non-word characters", even it does not exactly match `\w` only, but also excludes `-`?

Reply:
I have reworded the sentence to also explain the reason.